addons/snailmail/static/tests/chatter_snailmail_tests.js

```diff
@@ -106,10 +102,12 @@ QUnit.module('Chatter', {
 });
 
 QUnit.test('Sent', async function (assert) {
-    assert.expect(7);
+    assert.expect(6);
 
-    this.data['mail.message'].records[0].snailmail_status = 'sent';
-    this.data['mail.message'].records[0].snailmail_error = false;
+    this.data['mail.message'].records[0].notifications = [{
+        'notification_type': 'snailmail',
+        'notification_status': 'sent',
```

Comment:
no quotes around prop name when not necessary

Reply:
ok

Reply:
Actually not ok, kept as string since they are data that are "coming" from the server so they have to keep the snake case, but we would like to lint camel case some day ideally, which would fail in this case if it wasn't as string.